config.lua.dist

```diff
@@ -176,6 +176,7 @@ serverSaveNotifyDuration = 5
 serverSaveCleanMap = false
 serverSaveClose = false
 serverSaveShutdown = true
+serverSaveTimer = "06:00"
```

Comment:
Could change variable name to `serverSaveTime` and all of the others references.